compiler-core/src/javascript/tests/snapshots/gleam_core__javascript__tests__numbers__division_by_zero_int.snap

```diff
@@ -0,0 +1,15 @@
+---
+source: compiler-core/src/javascript/tests/numbers.rs
+expression: "pub fn main() {\n  1 / 0\n}"
+---
+----- SOURCE CODE
+pub fn main() {
+  1 / 0
```

Comment:
If we know it's 0 we could compile to just 0?

Reply:
Totally! I think in the long run it should end up being part of a constant folding step rather than special casing it during the code generation